web/src/bootstrap_typeahead.ts

```diff
@@ -665,7 +667,7 @@ export class Typeahead<ItemType extends string | object> {
                     // the search bar).
                     this.openInputFieldOnKeyUp();
                 }
-                this.lookup(false);
+                this.lookup(this.hideOnEmpty);
```

Comment:
I think naming this `hideOnEmpty` is a bit misleading, `hideOnEmptyDuringKeyup` or something would be more accurate.

But you should only be doing this for backspace, and call it `hideOnEmptyAfterBackspace`.

Reply:
Oh you mean something like this?
```
if (pseudo_keycode === 8){
   this.lookup(this.hideOnEmptyAfterBackspace);
   return;
}
```